mutation/mutation_partition.cc

```diff
@@ -325,10 +326,12 @@ stop_iteration mutation_partition::apply_monotonically(const schema& s, mutation
                 i = next_i;
             }
 
+            /*
             if (need_preempt() && i != end) {
                 res = apply_resume(stage, i->position());
                 return stop_iteration::no;
             }
+            */
```

Comment:
Why leave this commented code?

Reply:
I also don't understand why adding the new apply variants requires this change.

Reply:
sorry, this was left over unintentionally, while debugging.